frag-nav/src/main/java/com/ncapdevi/fragnav/FragNavController.kt

```diff
@@ -665,21 +689,16 @@ class FragNavController constructor(private val fragmentManger: FragmentManager,
                     }
                 }
 
-                setTransitionStyle(options.transitionStyle)
-
                 setTransition(options.transition)
 
                 options.sharedElements.forEach { sharedElement ->
-                    sharedElement.first?.let {
-                        sharedElement.second?.let { it1 ->
-                            addSharedElement(
-                                    it,
-                                    it1
-                            )
-                        }
-                    }
+                    addSharedElement(
+                        sharedElement.first,
```

Comment:
I think we can do this:
options.sharedElements.forEach { (sharedElement, sharedElementName) ->